backend/minors/AAS.yaml

```diff
@@ -0,0 +1,515 @@
+---
+type: Minor
+name: African American Studies
+code: AAS
+degree:
+description: Students in the Class of 2026 and beyond, who opt to pursue a minor in African American 
+  Studies (AAS) will gain access to an extraordinary bibliography that prepares them to think about 
+  race and power in sophisticated ways, expanding and deepening their understanding of race in the 
+  United States and in the world. Students are trained in the methods, themes and ideas that inform 
+  interdisciplinary scholarship, with a particular focus on race and racial inequality. 
+  Earning a minor is straightforward and allows students to experience an enriching course of study 
+  that complements any Princeton major.
+urls:
+- https://aas.princeton.edu/academics/undergraduate-program/minor-requirements
+contacts:
+- type: Director of Undergraduate Studies
+  name: Professor Keeanga-Yamahtta Taylor
+  email: aas21@princeton.edu
+req_list:
+- name: Core
+  max_counted: 1
+  min_needed: 2
+  explanation: Students must complete two AAS core survey courses. 
+  These courses serve as an introduction to the methods and 
+  areas of interdisciplinary study in the field. 
+  They are the cornerstone of the AAS minor, and thus there 
+  are no replacements for or exemptions from this 
+  requirement.
+  double_counting_allowed: false
+  max_common_with_major: 1 
+  pdfs_allowed: 1 # only one can count towards the minor
+  course_list:
+  - AAS 244
+  - AAS 245
+  - AAS 253
```

Comment:
Include the other "OR" options in

"AAS 244 Introduction to Pre-20th Century Black Diaspora Art
AAS 245 Introduction to 20th Century African American Art
AAS 253 Introduction to African American Literature to 1910 OR AAS 353 African American Literature: Origins to 1910 (pre-20th century)
AAS 254 Introduction to African American Literature since the Harlem Renaissance OR AAS 359 African American Literature: Harlem Renaissance to the Present
AAS 267 Introduction to African American History to 1863 OR AAS 366 African American History to 1863 (pre-20th century)
AAS 268 Introduction to African American History Since Emancipation OR AAS 367 African American Studies Since Emancipation"